deploy/sdk/src/dynamo/sdk/cli/containerize.py

```
@@ -0,0 +1,171 @@
+    Dynamo package and immediately building it into a Docker container.
+    """
+    try:
+        from dynamo.sdk.cli.build import BuildConfig, Package
+
+        if ":" not in service:
+            console.print(
+                "[red]Error: Service specification must be in format 'module:ServiceClass'[/]"
+            )
+            raise typer.Exit(1)
+
+        module_name, service_class = service.split(":", 1)
```

Pylint (3.3.7)

[warning] Unused variable 'module_name'

(W0612)